__tests__/index.tsx

```diff
@@ -148,6 +148,14 @@ describe('VideoRenderer', () => {
     xit('should return the buffered value', () => {
       // TODO: fake multiple ranges
     });
+
+    it('should return error status when the video is errored', () => {
+      const {component, children} = setup();
+
+      component.find('video').simulate('error');
+
+      expect(component.state('status')).toEqual('errored');
```

Comment:
Just a /nit for the future, we `state` admits a property, instead of `state().status` :)